expressServer/routes/auth.js

```diff
@@ -0,0 +1,19 @@
+import express from "express";
+import passport from "passport";
+import * as auth from '../controllers/auth';
+
+const router = express.Router();
+
+router.post("/", auth.auth);
+router.post("/local", auth.authPassport('local'));
+
+router.get("/twitter", passport.authenticate('twitter'));
```

Comment:
Move to a separate routes files

Reply:
@DraXu3 IMO it's logical to put all auth routes, both social-network and local to one file. There's no need to separate them

Reply:
It's not such important point to be resolved in the hometask's scope, so let's skip it. But I pay attention to the question: what type of grouping is more solid: per routes or per socials (and then re-think it with a case when we want to rename some social in future)?